src/api/status/public/js/utils/toast.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * UI theme for toast
+ * @typedef {("danger" | "success" | "info" | "warning" | "primary")} ToastVariant
+ */
+
+/**
+ * @returns {HTMLElement} the sample toast to clone from
+ */
+const querySampleToastEl = () => document.getElementById('sample-toast');
+/**
+ * @returns {HTMLElement} the container for all stacking toast elements
+ */
+const queryToastContainer = () => document.getElementById('toast-container');
+
+/**
+ * Make a new empty toast node from the sample toast node
+ *
+ * @returns {HTMLElement} an empty toast element
+ */
+const cloneDummyToastNode = () => {
```

Comment:
You use Element and Node in this file, and I wonder if you should standardize.  An Element becomes a node in the DOM when inserted, but I think here you are really returning an Element.

Reply:
You're right, I thought node and element are the same. When I read on MDN again, `Element` is a subclass of `Node`, and the `toast` is indeed an Element.